docs/system_overview/configuration.md

```diff
@@ -14,254 +14,682 @@ This scrambled password file is saved after an `iinit` is run. If this file does
 
 This file defines the behavior of the server Agent that answers individual requests coming into iRODS. It is created and populated by the installer package.
 
-This file contains the following top level entries:
+!!! IMPORTANT
+    All configuration settings are required unless marked as optional. The value associated with a key represents the default value. 
```

Comment:
I'm guessing these are the default values according to the schema?

Reply:
All defaults come from the template files.

I defined the value as an empty string or zero if there is no default defined in the template file.

Reply:
not according to the schema - just the server code itself.     and therefore, perhaps, the template.

Reply:
Ah, the templates. Thanks